internal/helpers.go

```diff
@@ -550,41 +551,41 @@ func FindStringInSlice(s []string, str string) bool {
 	return false
 }
 
-func ResetK1Dir(k1Dir string) error {
+func ResetK1Dir(k1Dir string, gitopsRepoName string, metaphorRepoName string) error {
 
 	if _, err := os.Stat(k1Dir + "/argo-workflows"); !os.IsNotExist(err) {
 		// path/to/whatever exists
 		err := os.RemoveAll(k1Dir + "/argo-workflows")
 		if err != nil {
-			return fmt.Errorf("unable to delete %q folder, error: %s", k1Dir+"/argo-workflows", err)
+			return fmt.Errorf("unable to delete %q folder, error: %w", k1Dir+"/argo-workflows", err)
 		}
 	}
 
-	if _, err := os.Stat(k1Dir + "/gitops"); !os.IsNotExist(err) {
-		err := os.RemoveAll(k1Dir + "/gitops")
+	if _, err := os.Stat(k1Dir + "/" + gitopsRepoName); !os.IsNotExist(err) {
+		err := os.RemoveAll(k1Dir + "/" + gitopsRepoName)
 		if err != nil {
-			return fmt.Errorf("unable to delete %q folder, error: %s", k1Dir+"/gitops", err)
+			return fmt.Errorf("unable to delete %q folder, error: %w", k1Dir+"/"+gitopsRepoName, err)
 		}
 	}
-	if _, err := os.Stat(k1Dir + "/metaphor"); !os.IsNotExist(err) {
-		err := os.RemoveAll(k1Dir + "/metaphor")
+	if _, err := os.Stat(k1Dir + "/" + metaphorRepoName); !os.IsNotExist(err) {
```

Comment:
You can also see a repeatable pattern, there are several lines with `k1Dir + "/" + metaphorRepoName` which could be merged into one declaration then the use of the variable.